get_invoice.go

```diff
@@ -0,0 +1,125 @@
+package fortnox
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/omniboost/go-fortnox/utils"
+)
+
+func (c *Client) NewGetInvoiceRequest() GetInvoiceRequest {
+	r := GetInvoiceRequest{
+		client:  c,
+		method:  http.MethodGet,
+		headers: http.Header{},
+	}
+
+	r.queryParams = r.NewGetInvoiceQueryParams()
+	r.pathParams = r.NewGetInvoicePathParams()
+	r.requestBody = r.NewGetInvoiceRequestBody()
+	return r
+}
+
+type GetInvoiceRequest struct {
+	client      *Client
+	queryParams *GetInvoiceQueryParams
+	pathParams  *GetInvoicePathParams
+	method      string
+	headers     http.Header
+	requestBody GetInvoiceRequestBody
+}
+
+func (r GetInvoiceRequest) NewGetInvoiceQueryParams() *GetInvoiceQueryParams {
+	return &GetInvoiceQueryParams{
+		// Pagination: odata.NewPagination(),
+	}
+}
+
+type GetInvoiceQueryParams struct{}
+
+func (p GetInvoiceQueryParams) ToURLValues() (url.Values, error) {
+	encoder := utils.NewSchemaEncoder()
+	params := url.Values{}
+
+	err := encoder.Encode(p, params)
+	if err != nil {
+		return params, err
+	}
+
+	return params, nil
+}
+
+func (r *GetInvoiceRequest) QueryParams() *GetInvoiceQueryParams {
+	return r.queryParams
+}
+
+func (r GetInvoiceRequest) NewGetInvoicePathParams() *GetInvoicePathParams {
+	return &GetInvoicePathParams{}
+}
+
+type GetInvoicePathParams struct {
+	DocumentNumber string
+}
+
+func (p *GetInvoicePathParams) Params() map[string]string {
+	return map[string]string{
+		"DocumentNumber": p.DocumentNumber,
+	}
+}
+
+func (r *GetInvoiceRequest) PathParams() *GetInvoicePathParams {
+	return r.pathParams
+}
+
+func (r *GetInvoiceRequest) SetMethod(method string) {
+	r.method = method
+}
+
+func (r *GetInvoiceRequest) Method() string {
+	return r.method
+}
+
+func (r GetInvoiceRequest) NewGetInvoiceRequestBody() GetInvoiceRequestBody {
+	return GetInvoiceRequestBody{}
+}
+
+type GetInvoiceRequestBody struct{}
+
+func (r *GetInvoiceRequest) RequestBody() *GetInvoiceRequestBody {
+	return &r.requestBody
+}
+
+func (r *GetInvoiceRequest) SetRequestBody(body GetInvoiceRequestBody) {
+	r.requestBody = body
+}
+
+func (r *GetInvoiceRequest) NewResponseBody() *GetInvoiceResponseBody {
+	return &GetInvoiceResponseBody{}
+}
+
+type GetInvoiceResponseBody struct {
+	MetaInformation `json:"MetaInformation"` // TODO: do we really get this?
```

Comment:
What's inside the meta info? 
Can you attach a printout and we'd then decide if we need it or now.